examples/util/setupbitrategraph.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const DataSeries = require('./timelinegraph').DataSeries;
+const GraphView = require('./timelinegraph').GraphView;
+
+/**
+ * Set up the bitrate graph for audio or video media.
+ * @param {string} kind - 'video' or 'audio'.
+ * @param {string} containerId - The id of the graph container.
+ * @param {string} canvasId - The id of the canvas.
+ */
+ function setupBitrateGraph(kind, containerId, canvasId) {
+  const bitrateSeries = new DataSeries();
+  const bitrateGraph = new GraphView(containerId, canvasId);
+
+  bitrateGraph.graphDiv_.style.display = 'none';
+  return async function startBitrateGraph(room, intervalMs) {
```

Comment:
Is `startBitrateGraph` async function? Looks like it need not be async function since it does not use `await` anywhere.  Because it declared as `async`, its return value is Promise<function> instead of `function` and that explains why using the return value as function gives an error that @timmydoza mentioned. 
```suggestion
  return function startBitrateGraph(room, intervalMs) {
```

Reply:
@makarandp0 it actually does since we do call `await room.getStats();`. Removing async will cause an error as well. I actually didn't get the same error that @timmydoza got without his suggested fix and my bitrate graph worked perfectly fine so I'm wondering why that must be as well.

Reply:
@PikaJoyce I think you just need function passed to setInterval to be async, you do not need `startBitrateGraph` to be async.